renderer/views/player.js

```diff
@@ -540,3 +548,37 @@ function cssBackgroundImageDarkGradient () {
   return 'radial-gradient(circle at center, ' +
     'rgba(0,0,0,0.4) 0%, rgba(0,0,0,1) 100%)'
 }
+
+// Format the playback time of the video
+function formatPlaybackProgress (currentTime, duration) {
```

Comment:
Change this function to just format a single time instead of both `currentTime` and `duration` since the code should be exactly the same for both. Then you can just call the function on `currentTime` and `duration` separately.